### DISCUSSION TITLE
Urgent Account Recovery Request for GitHub User: sagardhadke

### DISCUSSION THREAD

**Discussion by mobisoftad** [community/community#106523] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

Dear GitHub Team and all GitHub Staff,

I hope this message finds you well. I'm reaching out on behalf of a dear friend, Sagar Dhadke, who recently encountered an issue with their GitHub account. Approximately 5-6 days ago, they received a "Password Reset Notification Due to Suspicious Activity" email from GitHub.

Since then, they have diligently attempted to reset their password, only to be met with a message indicating that their account has been suspended, without any clear explanation. Understandably, this has caused significant concern and frustration for my friend, as their GitHub account, under the username sagardhadke, holds crucial repositories, totaling over 30.

Despite their efforts to resolve this issue by submitting 2-3 support tickets through your site, they have yet to receive any response or assistance from your team. This lack of communication and resolution has further exacerbated the situation for my friend.

I kindly implore the GitHub team to prioritize the recovery of my friend's account, sagardhadke, as soon as possible. Given the importance of the repositories stored within their account and the urgency of the matter, swift action is greatly appreciated.

Your prompt attention to this matter would not only alleviate my friend's distress but also reaffirm GitHub's commitment to supporting its users in times of need.

Thank you for your understanding and swift action in advance.

Warm regards,

Mobi Softad

![Email](https://github.com/community/community/assets/139642868/7cadfc50-b533-4ad1-b68c-f7d732830d16)
![account_suspended](https://github.com/community/community/assets/139642868/50d3ea13-10fc-438c-ac40-eca7118c6180)

**Comment by 11Cream** [comment]
![mail1](https://github.com/community/community/assets/27777800/8d100dd1-ff6b-42d5-8534-fe52934ed003)
![mail2](https://github.com/community/community/assets/27777800/9821b78d-3188-45b7-ac0c-bc4fa97eb492)
![mail3](https://github.com/community/community/assets/27777800/386d0bc0-a9f6-4122-b3c0-009aba8aa861)
![mail4](https://github.com/community/community/assets/27777800/0fb0d94f-d76e-45cd-ba1b-474ffee1c036)

![github](https://github.com/community/community/assets/27777800/de811e97-48db-4b24-91a1-1bc932bc4312)

Ticket ID: 2571360

same also  things also happen with my account

**Comment by mobisoftad** [comment]
@queenofcorgis Please Recover My friend Account As Soon As possible

**Reply by queenofcorgis** [reply]
Hi there,

Please note, contacting any member of GitHub Staff via unsolicited mentions or pings is strongly discouraged and may be considered a violation of our prohibition against harassment. Please review our Code of Conduct for more information.

Let's work together to keep GitHub Community a place where people feel safe to participate by being respectful of them and their time.

**Comment by queenofcorgis** [comment, marked as answer]
Hi @mobisoftad,

Thank you for your patience. The community is unable to assist with nor escalate your ticket, but we can assure you it is in the right place. It will be answered in the order it was received based on our team's resources and the amount of tickets ahead of yours. We appreciate your understanding!

As no other Community Discussions team members will be able to provide additional help with account related questions I am going to close this discussion.

Thank you!
